app/assets/static/picker.html

```diff
@@ -0,0 +1,42 @@
+<html>
```

Comment:
In the prototype version this HTML was included in the zip file which is why it includes the entire DOM. But as mentioned this code will get replaces when the files are generated and saved as a ZIP.